### PR TITLE
[WIP] Fix gdf to cudf

### PR DESCRIPTION
**Pull request by Iroy30**
access refactored cudf header files and libraries

### CODE REVIEW

File: include/cugraph.h [hunk 1]
```diff
@@ -3,9 +3,9 @@
 #include <cstdlib>
 #include <cstdint>
 
-#include <gdf/gdf.h>
-#include <gdf/utils.h>
-#include <gdf/errorutils.h>
+#include <cudf.h>
+#include <cudf_utils.h>
+#include <error_utils.h>
```
Comment (dantegd) [inline include/cugraph.h]: Has cuDF been patched to install this files?

Comment (Iroy30) [reply include/cugraph.h]: Not yet

Comment (Iroy30) [reply include/cugraph.h]: Hence the WIP

**Comment by Iroy30** [comment]
Fixed with #10

**Pull request closed without merging**